machine.go

```diff
@@ -518,14 +521,30 @@ func (m *Machine) startInstance(ctx context.Context) error {
 		ActionType: models.InstanceActionInfoActionTypeInstanceStart,
 	}
 
-	ctx, cancel := context.WithCancel(ctx)
-	defer cancel()
-
 	resp, err := m.client.CreateSyncAction(ctx, &info)
 	if err == nil {
 		m.logger.Printf("startInstance successful: %s", resp.Error())
 	} else {
-		m.logger.Printf("Error starting instance: %s", err)
+		m.logger.Errorf("Starting instance: %s", err)
```

Comment:
:+1: At some point I'll do an audit of all our messages and make sure the severities and language is correct. Thanks for taking care of this one.